src/pynxtools/__init__.py

```diff
@@ -24,8 +24,24 @@
 from pynxtools._build_wrapper import get_vcs_version
 from pynxtools.definitions.dev_tools.globals.nxdl import get_nxdl_version
 
+
+class CustomFormatter(logging.Formatter):
+    """Formatter that specifically highlights errors and warnings."""
+
+    def format(self, record):
+        if not getattr(record, "prefixed", False):
+            if record.levelno in (logging.WARNING, logging.ERROR):
```

Comment:
Maybe this can be made a constant and be specified on top of the file. Then it's easy to see and change on which levels the level name will be displayed.

Reply:
Sounds good, implemented